api/src/pdc/services/trip/actions/PublishOpenDataAction.ts

```
@@ -24,45 +20,14 @@ import { GetRessourceIdIfExists } from "./opendata/GetRessourceIdIfExists.ts";
   ]],
 })
 export class PublishOpenDataAction extends Action {
-  constructor(
-    private config: ConfigInterfaceResolver,
-    private datagouv: DataGouvProvider,
-    private buildResourceDescription: BuildResourceDescription,
-    private getRessourceIdIfExists: GetRessourceIdIfExists,
-  ) {
+  constructor() {
     super();
   }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)